llvm/lib/CodeGen/AsmPrinter/AccelTable.cpp

```diff
@@ -358,8 +359,13 @@ void AppleAccelTableWriter::emit() const {
 }
 
 DWARF5AccelTableData::DWARF5AccelTableData(const DIE &Die,
-                                           const DwarfCompileUnit &CU)
-    : OffsetVal(&Die), DieTag(Die.getTag()), UnitID(CU.getUniqueID()) {}
+                                           const DwarfUnit &Unit,
+                                           const bool IsTU)
+    : OffsetVal(&Die) {
+  Data.DieTag = Die.getTag();
+  Data.UnitID = Unit.getUniqueID();
+  Data.IsTU = IsTU;
```

Comment:
If these members weren't in a nested struct (see comment on this `Data` member) - these maybe could move to the ctor init list